src/components/LayerOpacityPicker.vue

```diff
@@ -0,0 +1,119 @@
+<!--
+  @file
+  @since v3.8.0
+-->
+
+<template>
+
+  <div
+    @mouseleave.self="showLayerOpacityMenu(false, $event)"
+    @mouseover.self="showLayerOpacityMenu(true, $event)"
+  >
```

Comment:
Perhaps the sub-context menu positioning should be handled within the `CatalogLayerContextMenu.vue` component. But for now I think it's okay too.

Reply:
@Raruto i agree. Maybe when we refactor all CatalogLayerComponentMenu in an isolate vue componet we handle it